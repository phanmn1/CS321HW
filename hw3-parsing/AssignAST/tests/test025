i=j || k==l
// valid, but not parsed as layout suggests!
