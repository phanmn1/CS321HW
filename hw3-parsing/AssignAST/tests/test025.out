Assign
  "i"
  LOr
    Id("j")
    Eql
      Id("k")
      Id("l")
